build.gradle

```diff
@@ -160,50 +212,48 @@ task zip(type: Zip) {
     }
 }
 
-task installPlugin() {
-    if (!new File("${rootDir}/dist").exists()) {
-        return
+tasks.register('installPlugin') {
+    // Compute dependency and outputs eagerly
+    dependsOn('dist')
```

Comment:
I think `installPlugin` doesn't need to depend on `dist` or `jar`, according to https://github.com/apache/eventmesh/pull/4719#discussion_r1554661834.

Depending on `dist` will significantly lengthen the time of task execution. So the original `exists()` check will be better.

Reply:
Fixed